### PR TITLE
Handle reader-mode tabs, by storing an extra bit of info in the message.

### LINKED ISSUE #424: Can't snooze a tab in read mode
When a tab is in read mode, the snooze icon is grayed and I can't snooze it. 
I have to leave the read mode.

### PR DESCRIPTION
**Pull request by bwinton**
Fixes #424.
@rugk, @gaul: Either of you want to take a look at this and give it a 👍🏼/👎🏼?

### CODE REVIEW

**Review by rugk — APPROVED** [review]
Code-wise this LGTM, but what does thus achieve?

Could not you (previously) just re-open tabs in reader mode as they were… in reader mode again. Or does not this work?

**Edit:** Ah forget it, you do that, there is a `openInReaderMode` flag, awesome! That looks nice. :+1:

File: src/popup/snooze-content.js [hunk 1]
```diff
@@ -53,9 +54,11 @@ function undeleteSnoozedTab(item) {
 }
 
 function openSnoozedTab(item) {
+  log('openSnoozedTab', item);
```
Comment (rugk) [inline src/popup/snooze-content.js:57]: Is this log intentional or remaining from debugging?

**Comment by rugk** [comment]
BTW watched PRs now, so you don't need to ping me everytime… :wink:

File: src/popup/snooze-content.js (hunk 1 above)
Comment (bwinton) [reply src/popup/snooze-content.js:57]: Whoops! Removed!

**Pull request merged**